src/main/routes/task-list.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express';
+import { SubTaskListBuilder } from '../app/case/util/subTaskListBuilder';
+
+const router = express.Router();
+
+router.get('/task-list', (req, res) => {
+  res.render('task-list', {
```

Comment:
instead of putting view name here, we will derive that from Path name. ok for now but this has to be updated too.